packages/seccomp-tools/PKGBUILD

```
@@ -0,0 +1,85 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=seccomp-tools
+pkgver=v1.6.1.r36.g728ec2f
+ver="${pkgver%%.r*}" 
+pkgrel=1
+pkgdesc="Provide powerful tools for seccomp analysis."
+arch=('any')
+groups=('blackarch' 'blackarch-binary' 'blackarch-decompiler' 'blackarch-reversing')
+url="https://github.com/david942j/seccomp-tools"
+license=('MIT')
+depends=('glibc' 'ruby' 'ruby-os' 'ruby-racc')
```

```suggestion
depends=('glibc' 'ruby' 'ruby-os')
```

ruby-racc looks like a build dependency to me https://github.com/search?q=repo%3Adavid942j%2Fseccomp-tools%20racc&type=code

---

I tried to test it without `ruby-racc` but when I launch the tool, I get this error:
```
seccomp-tools

<internal:/usr/lib/ruby/3.4.0/rubygems/core_ext/kernel_require.rb>:136:in 'Kernel#require': cannot load such file -- racc/parser.rb (LoadError)
	from <internal:/usr/lib/ruby/3.4.0/rubygems/core_ext/kernel_require.rb>:136:in 'Kernel#require'
	from /usr/lib/ruby/gems/3.4.0/gems/seccomp-tools-1.6.1/lib/seccomp-tools/asm/sasm.tab.rb:7:in '<top (required)>'
	from <internal:/usr/lib/ruby/3.4.0/rubygems/core_ext/kernel_require.rb>:136:in 'Kernel#require'
	from <internal:/usr/lib/ruby/3.4.0/rubygems/core_ext/kernel_require.rb>:136:in 'Kernel#require'
	from /usr/lib/ruby/gems/3.4.0/gems/seccomp-tools-1.6.1/lib/seccomp-tools/asm/compiler.rb:3:in '<top (required)>'
	from <internal:/usr/lib/ruby/3.4.0/rubygems/core_ext/kernel_require.rb>:136:in 'Kernel#require'
	from <internal:/usr/lib/ruby/3.4.0/rubygems/core_ext/kernel_require.rb>:136:in 'Kernel#require'
	from /usr/lib/ruby/gems/3.4.0/gems/seccomp-tools-1.6.1/lib/seccomp-tools/asm/asm.rb:3:in '<top (required)>'
	from <internal:/usr/lib/ruby/3.4.0/rubygems/core_ext/kernel_require.rb>:136:in 'Kernel#require'
	from <internal:/usr/lib/ruby/3.4.0/rubygems/core_ext/kernel_require.rb>:136:in 'Kernel#require'
	from /usr/lib/ruby/gems/3.4.0/gems/seccomp-tools-1.6.1/lib/seccomp-tools/cli/asm.rb:4:in '<top (required)>'
	from <internal:/usr/lib/ruby/3.4.0/rubygems/core_ext/kernel_require.rb>:136:in 'Kernel#require'
	from <internal:/usr/lib/ruby/3.4.0/rubygems/core_ext/kernel_require.rb>:136:in 'Kernel#require'
	from /usr/lib/ruby/gems/3.4.0/gems/seccomp-tools-1.6.1/lib/seccomp-tools/cli/cli.rb:3:in '<top (required)>'
	from <internal:/usr/lib/ruby/3.4.0/rubygems/core_ext/kernel_require.rb>:136:in 'Kernel#require'
	from <internal:/usr/lib/ruby/3.4.0/rubygems/core_ext/kernel_require.rb>:136:in 'Kernel#require'
	from /usr/lib/ruby/gems/3.4.0/gems/seccomp-tools-1.6.1/bin/seccomp-tools:4:in '<top (required)>'
	from /usr/bin/seccomp-tools:25:in 'Kernel#load'
	from /usr/bin/seccomp-tools:25:in '<main>'
```
Could be actually needed for runtime?

---

I don't think it's used at runtime, but lib/seccomp-tools.rb requires lib/seccomp-tools/asm/asm.rb which requires lib/seccomp-tools/asm/compiler.rb which requires lib/seccomp-tools/asm/sasm.tab.rb which requires racc. So even if unused at runtime because it's transitively required from the root level the dependency must be provided.